### PR TITLE
test: add a test for the web build

### PR DESCRIPTION
**Pull request by OrKoN**

### CODE REVIEW

File: test/installation/assets/puppeteer/puppeteer-in-browser.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+const browser = await puppeteer.launch({
+  headless: true,
+  args: ['--remote-allow-origins=*'],
+});
+
+try {
+  const port = process.argv[2];
+
+  console.log(port);
```
Comment (Lightning00Blade) [inline test/installation/assets/puppeteer/puppeteer-in-browser.js]: Remove?

File: test/installation/assets/puppeteer/puppeteer-in-browser.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  await page.type('input', browser.wsEndpoint());
+
+  const result = new Promise(resolve => {
+    page.once('dialog', dialog => {
+      dialog.accept();
+      resolve(dialog.message());
+    });
+  });
+
+  await page.click('button');
```
Comment (Lightning00Blade) [inline test/installation/assets/puppeteer/puppeteer-in-browser.js]: Let's use Promise.all here it's better practice.

File: test/installation/src/puppeteer.spec.ts [hunk 3]
```diff
@@ -37,6 +42,34 @@ describe('`puppeteer`', () => {
...
+  it('runs in the browser', async function () {
+    const puppeteerInBrowserPath = join(this.sandbox, 'puppeteer-in-browser');
+    fs.cpSync(
+      join(EXAMPLES_DIR, 'puppeteer-in-browser'),
+      puppeteerInBrowserPath,
+      {
+        recursive: true,
+      }
+    );
+    console.log(this.sandbox);
```
Comment (Lightning00Blade) [inline test/installation/src/puppeteer.spec.ts]: Remove log.

**Review by Lightning00Blade — APPROVED** [review]
LGTM + small comments

File: test/installation/assets/puppeteer/puppeteer-in-browser.js (hunk 2 above)
Comment (OrKoN) [reply test/installation/assets/puppeteer/puppeteer-in-browser.js]: why is it better?

Comment (Lightning00Blade) [reply test/installation/assets/puppeteer/puppeteer-in-browser.js]: It's more of a pattern to use/show users: Set up listener and execute the command at the same time.
I acknowledge that this is correct as is it.

**Pull request merged**